django_loci/tests/base/test_geocode.py

```diff
@@ -0,0 +1,63 @@
+from django.urls import reverse
+
+from .. import TestAdminMixin
+
+
+class BaseTestGeocode(TestAdminMixin):
+    def test_geocode(self):
```

Comment:
please move these tests to `test_admin` so that we avoid having to change openwisp-controller.